File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        byte []arr = new byte[nums.length+1];
        for(int i: nums){
            arr[i] = 1;
        }
        for(int i = 0; i < nums.length; i++){
            if(arr[i] == 0) return i;
        }
        return nums.length;
    }
}

File: 268-missing-number/missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        nums.sort()
        l = len(nums)
        if nums[l-1]==l-1:
            return l
        for i in range(len(nums)):
            if nums[i]!=i:
                return nums[i]-1